src/com/company/Main.java

```diff
@@ -37,76 +47,158 @@ public static void main(String[] args) {
                     searchContact();
 
                 case 5:
-                    printActions();
+                    questions();
                     break;
 
-//                case 6:
-//                    questions();
-//                    break;
+                case 6:
+                    printOptions();
+                    break;
             }
         }
 
     }
 
+    private static void printOptions() {
+        System.out.println("\nCreate your profile");
+        System.out.println("\t 0 to quit");
+        System.out.println("\t 1 to show persons");
+        System.out.println("\t 2 to add new person");
+        System.out.println("\t 3 to remove person");
+        System.out.println("\t 4 to search person");
+        System.out.println("\t 5 answer our matching questions");
+        System.out.println("\t 6 to show options");
+        System.out.println("Choose your option: ");
+    }
+
     private static void createPerson() {
-        System.out.println("Enter your full name");
-        String fullName = scanner.nextLine();
-        System.out.println("Enter your gender(0 for male, 1 for female)");
-        int gender = Integer.parseInt(scanner.nextLine());
+
+        System.out.println("Enter your name");
+        String name = scanner.nextLine();
+        if (name.matches("[a-zA-z]{3,}")) {
```

Comment:
Thats what im talking about! Now extract it in an constant(private static final String)